env0/resource_configuration_variable.go

```diff
@@ -110,6 +110,11 @@ func resourceConfigurationVariable() *schema.Resource {
 				Description: "the value of this variable must match provided regular expression (enforced only in env0 UI)",
 				Optional:    true,
 			},
+			"soft_delete": {
```

Comment:
@TomerHeber now that I think about it we could have added it to "environment import resource" as well so that it won't be implicit that its not actually deleted